### PR TITLE
Added Autostart via GUI

### PR DESCRIPTION
**Pull request by TongruiLi**
Start VESC/LiDAR/Joystick/Websocket when GUI Icon is clicked.

### CODE REVIEW

File: src/gui/gui_mainwindow.cc [hunk 1]
```diff
@@ -322,16 +322,16 @@ void Exec(const string& cmd) {
...
 
 void MainWindow::StartRos() {
   const string path = ros::package::getPath("ut_automata");
   Exec("/usr/bin/screen -mdS roscore roscore");
 }
 
-void MainWindow::StopRos() {
-  Exec("/usr/bin/killall roscore");
+void MainWindow::StartCamera() {
+  Exec("roslaunch astra_camera astra.launch");
```
Comment (joydeep-b) [inline src/gui/gui_mainwindow.cc:332]: This calls the script that also starts the GUI

File: scripts/start_car_gui.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```
Comment (joydeep-b) [inline scripts/start_car_gui.sh]: Why do we need another start script?

File: gui/gui.desktop [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
```
Comment (joydeep-b) [inline gui/gui.desktop]: This can go under `scripts`.

**Review by joydeep-b — CHANGES_REQUESTED** [review]
1. The start button on the GUI calls the script that also tries to start the GUI (again).
2. Why do we need another start script?

File: src/gui/gui_mainwindow.cc [hunk 4]
```diff
@@ -321,12 +321,10 @@ void Exec(const string& cmd) {
 }
 
 void MainWindow::StartCar() {
-  const string path = ros::package::getPath("ut_automata");
-  Exec(path + "/scripts/start_car_gui.sh");
+  Exec("roslaunch ut_automata start_car.launch start_gui:=flase");
```
Comment (TongruiLi) [inline src/gui/gui_mainwindow.cc:324]: Need to fix typo.

**Pull request merged**